specs/vep-994-jupyter-notebook-integration/README.md

```diff
@@ -129,16 +176,25 @@ Example use case:
 
 ![example-use-case](example.png)
 
+### Availability
+The availability of the extension will be managed by JupyterLab since it is going to be run as part of the JupyterLab ecosystem.
 
+### Test Plan
+* The front-end extension is using Jest for JavaScript code testing.
+* This extension uses Playwright for the integration tests (aka user level tests). More precisely, the JupyterLab helper Galata is used to handle testing the extension in JupyterLab.
+* The server extension is using Pytest for Python code testing.
+* The VDK notebook plugin will be using Pytest for Python code testing.
+
+### Security and Permissions
+In terms of security, Jupyter uses tornado to ensure only authorized user can request the Jupyter server
```

Comment:
You can add a todo only but a few more security concerns need to be looked into 
* What would be the authenticaiton/authorizaiton against the control service  -you can refer to control service doc for this.